src/Cartography.DynamicMap/MapControlBase.cs

```diff
@@ -511,6 +513,7 @@ private async Task TrySetViewPort(CancellationToken ct, MapViewPort viewPort)
         }
         catch (Exception ex)
         {
+            _logger.LogError(ex.Message);
```

Comment:
You shouldn't need this. I think the exception needs to be the first parameter of the LogError method.